Misc/NEWS.d/next/Library/2021-11-08-20-27-41.bpo-44439.I_8qro.rst

```diff
@@ -0,0 +1,2 @@
+Fix in `_ZipWriteFile.write()` method, when the input data is an object that
```

Comment:
`_ZipWriteFile` is a private class. It would be better to reword the NEWS entry in terms of the public API.